src/core/Normalization.helper.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-plusplus */
+/* eslint no-await-in-loop: 0 */
+export default class NormalizationHelper {
+    /**
+     *
+     * @param {String|Number} value  - The value that will be normalized
+     * @param {Boolean} fullUnit - This is by default on true. If you put this on false, the method will not do the multiplication
+     * @returns {Number}
+     */
+    static normalizeGoalValue (value, fullUnit) {
+
+        if (fullUnit !== false) {
+            fullUnit = true;
+        }
+
+        let normalized = parseFloat(value);
+        if (isNaN(normalized)) {
+            return 0;
+        } else if (normalized % 1 === 0) {
+            if (fullUnit) {
+                return normalized * 100;
+            } else {
+                return normalized;
+            }
+        } else {
+            if (fullUnit) {
+                normalized = normalized.toFixed(2);
```

Comment:
[INFO] why don't we convert this to Number as we do on line 30?

Reply:
I will add that conversion, but the "* 100" underneath, converts it to a number implicitly but I agree that adding a specific cast is good for clarity and to avoid edge cases.